usaspending_api/api_contracts/contracts/v2/search/spending_by_award.md

```diff
@@ -195,6 +195,7 @@ List of table columns
 + `treasury_account_components` (optional, array[TreasuryAccountComponentsObject], fixed-type)
 + `object_class` (optional, array[string])
 + `program_activity` (optional, array[number])
++ `def_codes` (optional, array[string])
```

Comment:
On the disaster endpoints we use an enum for def_codes: https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/disaster/award/spending.md#filter-object

Might as well copy it  over here.

Reply:
so be it